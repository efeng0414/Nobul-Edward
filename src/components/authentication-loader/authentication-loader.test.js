import React from "react";
import renderer from "react-test-renderer";
import AuthenticationLoader from "./index";

describe("AuthenticationLoader", () => {
  const componentJSX = <AuthenticationLoader type="loading" />;

  describe("Component Snapshot", () => {
    const componentSnapshot = renderer.create(componentJSX).toJSON();
    it("should match existing snapshot", () => {
      expect(componentSnapshot).toMatchSnapshot();
    });
  });
});
