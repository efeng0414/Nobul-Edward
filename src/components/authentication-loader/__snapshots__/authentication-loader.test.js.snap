// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthenticationLoader Component Snapshot should match existing snapshot 1`] = `
<div
  className="ant-spin-nested-loading spinner-wrapper"
  style={null}
>
  <div>
    <div
      className="ant-spin ant-spin-lg ant-spin-spinning ant-spin-show-text"
    >
      <span
        className="ant-spin-dot ant-spin-dot-spin"
      >
        <i />
        <i />
        <i />
        <i />
      </span>
      <div
        className="ant-spin-text"
      >
        Loading...
      </div>
    </div>
  </div>
  <div
    className="ant-spin-container ant-spin-blur"
  >
    <div
      className="loading-page"
    />
  </div>
</div>
`;
